src/common/bfloat16.hpp

```diff
@@ -38,6 +38,11 @@ struct bfloat16_t {
 
     DNNL_API operator float() const;
 
+    bfloat16_t &operator+=(const float a) {
+        (*this) = float{*this} + a;
+        return *this;
+    }
+
```

Comment:
(random place)

It seems clang format is not happy with the patch (see the CI). Not sure why though...
Could you please handle this?

Reply:
Thanks!